Library/Formula/qjackctl.rb

```diff
@@ -0,0 +1,118 @@
+class Qjackctl < Formula
+  desc "simple Qt application to control the JACK sound server daemon"
+  homepage "http://qjackctl.sourceforge.net"
+  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
+  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
+
+  depends_on "autoconf" => :build
+  depends_on "qt"
+  depends_on "jack"
+
+  patch :DATA
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-qt4",
+                          "--disable-dbus",
+                          "--disable-xunique",
+                          "--prefix=#{prefix}"
+
+    system "make"
+    prefix.install "src/qjackctl.app"
+    man1.install "qjackctl.1"
```

Comment:
There is a make install, but it is not of much help: it installs the app to ``/usr/local/Cellar/qjackctl/0.4.1/bin/qjackctl.app/`` which is not what we want I guess.
Also when using make install, I don't know how to make homebrew show up the ``Run `brew linkapps qjackctl` to symlink these to /Applications.``-message